cli/azd/pkg/input/console.go

```diff
@@ -591,31 +580,79 @@ func (c *AskerConsole) Handles() ConsoleHandles {
 	return c.handles
 }
 
-const cMinConsoleWidth int = 40
-
 func getConsoleWidth() int {
 	width, _ := consolesize.GetConsoleSize()
-	if width < cMinConsoleWidth {
-		return cMinConsoleWidth
-	}
 	return width
 }
 
+func (c *AskerConsole) handleResize(width int) {
+	c.consoleWidth.Store(int32(width))
+
+	c.spinnerLineMu.Lock()
+	if c.spinner.Status() == yacspin.SpinnerRunning {
+		line := c.spinnerLine(c.spinnerCurrentTitle, c.currentIndent.Load())
+		c.spinner.Message(line.Message)
+		_ = c.spinner.CharSet(line.CharSet)
+		c.spinner.Prefix(line.Prefix)
+	}
+	c.spinnerLineMu.Unlock()
```

Comment:
just wondering - could we defer this call right after we do the lock, so it's less likely that if we add an early return we forget to unlock?

Reply:
The way this is currently written, where `handleResize` ends up being a higher-level function, the `defer` may end up holding on the mutex longer than necessary, keeping it as-is for now.